### PR TITLE
Refactoring argparse

### PR DESCRIPTION
**Pull request by MrSyee**
Refactor argparse
- Remove argparse object of argument of class.
- Unpack argparse and input argument of class.
- This is to ensure that factors are not hidden in argparse or dictionary.

### CODE REVIEW

File: rl_algorithms/common/env/utils.py [hunk 1]
```diff
@@ -31,18 +30,18 @@ def set_env(
...
+    env_name: str, max_episode_steps: int, env_wrappers: List[gym.Wrapper] = None
 ) -> Callable:
     """Return env creating function (with normalizers)."""
 
     def _thunk(rank: int):
         env = gym.make(env_name)
-        env.seed(args.seed + rank + 1)
-        env = set_env(env, args, env_wrappers)
+        env.seed(777 + rank + 1)
+        env = set_env(env, max_episode_steps, env_wrappers)
```
Comment (jinPrelude) [inline rl_algorithms/common/env/utils.py:44]: Receiving max_episode_step explicitly might looks better.

Comment (MrSyee) [reply rl_algorithms/common/env/utils.py]: I was missing this issue. I've fixed it.

File: rl_algorithms/common/env/utils.py [hunk 2]
```diff
@@ -31,18 +30,18 @@ def set_env(
...
-    env_name: str, args: argparse.Namespace, env_wrappers: List[gym.Wrapper] = None
+    env_name: str, max_episode_steps: int, env_wrappers: List[gym.Wrapper] = None
 ) -> Callable:
     """Return env creating function (with normalizers)."""
 
     def _thunk(rank: int):
         env = gym.make(env_name)
-        env.seed(args.seed + rank + 1)
-        env = set_env(env, args, env_wrappers)
+        env.seed(777 + rank + 1)
```
Comment (jinPrelude) [inline rl_algorithms/common/env/utils.py:43]: seed is fixed.

File: rl_algorithms/common/abstract/agent.py [hunk 3]
```diff
@@ -71,16 +86,21 @@ def train(self):
     def set_wandb(self):
         """Set configuration for wandb logging."""
         wandb.init(
-            project=self.log_cfg.env_name,
+            project=self.env_info.name,
             name=f"{self.log_cfg.agent}/{self.log_cfg.curr_time}",
         )
-        wandb.config.update(vars(self.args))
+        added_log = dict(
```
Comment (jiseongHAN) [inline rl_algorithms/common/abstract/agent.py:92]: 변수명을 바꾸어도 좋을 것 같습니다

Comment (MrSyee) [reply rl_algorithms/common/abstract/agent.py:92]: 어떤게 좋을까요?

Comment (jinPrelude) [reply rl_algorithms/common/abstract/agent.py:92]: additional_log

**Review by jiseongHAN — APPROVED** [review]

File: rl_algorithms/common/env/utils.py (hunk 2 above)
Comment (jinPrelude) [reply rl_algorithms/common/env/utils.py:43]: Since it's hard to parse environment seed in ppo setting We decided to leave it for the moment.

**Review by jinPrelude — APPROVED** [review]
